src/main/telemetry/smartport.c

```diff
@@ -352,15 +341,11 @@ bool canSendSmartPortTelemetry(void)
 
 void checkSmartPortTelemetryState(void)
 {
-    bool newTelemetryEnabledValue = (smartPortPortSharing == PORTSHARING_NOT_SHARED);
-    if (newTelemetryEnabledValue == smartPortTelemetryEnabled) {
+    if (!smartPortTelemetryEnabled) {
```

Comment:
Won't that prevent smart port from being configured ever? `smartPortTelemetryEnabled` is set in the configure function only. I believe the same applies to HoTT as well.